frontends/user-tutorials/sidebar/components/PublishTutorialModal/PublishTutorialModal.tsx

```diff
@@ -73,9 +77,13 @@ export const PublishTutorialModal: FC = () => {
     publish(formData);
   };
 
+  useEffect(() => {
```

Comment:
you should catch the error inside submitHandler. Otherwise it will cause one more render, because you are handling error state in useEffect